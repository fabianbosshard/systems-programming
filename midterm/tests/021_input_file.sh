exec "$1" format='Option @a has value @b' delim== tests/lcov_config
