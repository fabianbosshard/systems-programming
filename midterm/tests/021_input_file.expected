Option genhtml_css_file  has value  gcov.css
Option genhtml_dark_mode  has value  1
Option genhtml_header  has value  Coverage report for my project
Option genhtml_footer  has value  My footer text
Option genhtml_hi_limit  has value  90
Option genhtml_med_limit  has value  75
Option ignore_errors  has value  empty,mismatch
Option max_message_count  has value  100
Option stop_on_error  has value  1
Option rtl_file_extensions  has value  v,vh,sv,vhd?
Option c_file_extensions  has value  h,c,i,C,H,I,cpp,hpp,icc,cc,hh,cxx,hxx
Option filter_lookahead  has value  5
Option filter_bitwise_conditional  has value  1
Option genhtml_line_field_width  has value  12
Option genhtml_branch_field_width  has value  16
Option genhtml_owner_field_width  has value  20
Option genhtml_age_field_width  has value  5
Option genhtml_overview_width  has value  80
Option genhtml_nav_resolution  has value  4
Option genhtml_nav_offset  has value  10
Option genhtml_keep_descriptions  has value  0
Option genhtml_no_prefix  has value  0
Option genhtml_no_source  has value  0
Option genhtml_num_spaces  has value  8
Option genhtml_highlight  has value  0
Option genhtml_legend  has value  0
Option genhtml_html_prolog  has value  FILE
Option genhtml_html_epilog  has value  FILE
Option genhtml_html_extension  has value  html
Option genhtml_html_gzip  has value  1
Option genhtml_sort  has value  1
Option genhtml_hierarchical  has value  1
Option genhtml_flat_view  has value  1
Option genhtml_show_navigation  has value  1
Option genhtml_show_function_proportion  has value  0
Option genhtml_charset has value UTF-8
Option genhtml_desc_html has value 0
Option genhtml_precision has value 1
Option genhtml_missed has value 1
Option suppress_function_aliases  has value  1
Option derive_function_end_line  has value  1
Option derive_function_end_line  has value  1
Option trivial_function_threshold  has value  5
Option demangle_cpp  has value  c++filt
Option genhtml_demangle_cpp_tool  has value  c++filt
Option genhtml_demangle_cpp_params  has value  ""
Option geninfo_gcov_tool  has value  gcov
Option geninfo_adjust_testname  has value  0
Option forget_testcase_names  has value  0
Option checksum  has value  1
Option geninfo_external  has value  1
Option geninfo_compat_libtool  has value  0
Option geninfo_gcov_all_blocks  has value  1
Option geninfo_unexecuted_blocks  has value  0
Option geninfo_compat  has value  libtool
Option geninfo_adjust_src_path  has value  /tmp/build 
Option geninfo_auto_base  has value  1
Option geninfo_intermediate  has value  auto
Option no_exception_branch  has value  0
Option lcov_insmod_tool  has value  /sbin/insmod
Option lcov_modprobe_tool  has value  /sbin/modprobe
Option lcov_rmmod_tool  has value  /sbin/rmmod
Option lcov_tmp_dir  has value  /tmp
Option lcov_list_full_path  has value  0
Option lcov_list_width  has value  80
Option lcov_list_truncate_max  has value  20
Option function_coverage  has value  1
Option branch_coverage  has value  0
Option lcov_fail_under_lines  has value  97.5
Option genhtml_annotate_script  has value  path_to_my_executable
Option genhtml_criteria_script  has value  path_to_my_executable
Option criteria_callback_data  has value  date,owner
Option criteria_callback_levels  has value  top,directory,file
Option genhtml_date_bins  has value  7
Option genhtml_date_bins  has value  30,180
Option include  has value  my/files/*.c
Option include  has value  my/files/*.h
Option exclude  has value  your/files/*.c
Option exclude  has value  your/files/*.h
Option substitute  has value  s#/.libs##g
Option substitute  has value  s#/tmp/build#/usr/src#g
Option omit_lines  has value  .+?//\s*MY_LINE_EXCLUDE_MARKER\s*
Option omit_lines  has value  .+?//\s*MY_PROJECT_EXCLUDE_MARKER\s*
Option erase_functions  has value  main
Option erase_functions  has value  debug)_.*\(int,\s*int\)
Option version_script  has value  path_to_my_executable
Option lcov_json_module  has value  auto
Option parallel  has value  1
Option memory  has value  1024
Option split_char  has value  ,
Option case_insensitive  has value  0
Option lcov_excl_line  has value  LCOV_EXCL_LINE
Option lcov_excl_br_line  has value  LCOV_EXCL_BR_LINE
Option lcov_excl_exception_br_line  has value  LCOV_EXCL_EXCEPTION_BR_LINE
Option lcov_excl_start  has value  LCOV_EXCL_START
Option lcov_excl_stop  has value  LCOV_EXCL_STOP
Option lcov_excl_br_start  has value  LCOV_EXCL_BR_START
Option lcov_excl_br_stop  has value  LCOV_EXCL_BR_STOP
Option lcov_excl_exception_br_start  has value  LCOV_EXCL_EXCEPTION_BR_START
Option lcov_excl_exception_br_stop  has value  LCOV_EXCL_EXCEPTION_BR_STOP
